src/Microsoft.PowerApps.TestEngine/SingleTestRunner.cs

```diff
@@ -236,6 +236,8 @@ public async Task RunTestAsync(string testRunId, string testRunDirectory, TestSu
             finally
             {
                 // Trying to log the debug info including session details
+                // Consider avoiding calling DebugInfo in cases where the PowerAppsTestEngine object is not needed
```

Comment:
I do not think we should leave this comment here. Essentially this is a TODO? Maybe if so, we need to handle it soon.